src/anyon/qpu_interface.jl

```diff
@@ -61,11 +64,14 @@ function post_request(
     body::String,
 )::HTTP.Response
 
+    package_version = string(Pkg.project().version)
```

Comment:
I don't think this does what we want it to do. If someone is in another project, such a `QBenchmark`, then `project().version` will return `QBenchmarks` version instead of `Snowlfurrys` version. The only reliable way to determine a package version is with `pkgversion(@__MODULE__)`, but `pkgversion` is only supported since Julia 1.9. Prior to that there are ways that rely on non-public interfaces or parsing the TOML file of the package in question. We should either drop support for Julia 1.8 and earlier, or go the TOML parsing route.

Reply:
updated to read from Project.toml